getting-started/get-the-sdk.md

```diff
@@ -53,55 +53,8 @@ You should see a pop-up similar to the following \(image below shows iOS\):
 For React Native, it is recommended to first install [Node.js](https://nodejs.org) to download packages from [npm](https://npmjs.com). See this [tutorial](https://facebook.github.io/react-native/docs/getting-started) for additional instructions on getting started with React Native applications
 {% endhint %}
 
-`@adobe/react-native-acpcore` is a wrapper around the iOS and Android Adobe Experience Platform Mobile SDK to allow for integration with React Native applications. Functionality to enable the Core extension is provided entirely through JavaScript documented below.
+See the README in the [react-native-acpcore](https://github.com/adobe/react-native-acpcore) repository for the latest React Native installation instructions.
```

Comment:
Please rephrase as follows:
"For the latest React Native installation instructions, see the README file in the [react-native-acpcore](https://github.com/adobe/react-native-acpcore) repository ."